<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EthioSurvRiskToolboxDialogBase</class>
 <widget class="QDialog" name="EthioSurvRiskToolboxDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EthioSurv-RiskToolbox</string>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <item>
    <widget class="QTabWidget" name="tab_widget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_risk_analysis">
      <attribute name="title">
       <string>1. Risk Analysis</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_project_def">
         <property name="text">
          <string><b>Project Definition</b></string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_project_name">
         <property name="text">
          <string>Project Name:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_project_name"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_objective">
         <property name="text">
          <string>Surveillance Objective:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="combo_objective"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_study_area">
         <property name="text">
          <string>Study Area Layer:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QgsMapLayerComboBox" name="mMapLayerComboBox_study_area"/>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="label_risk_factors">
         <property name="text">
          <string><b>Risk Factor Modeling</b></string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QTableWidget" name="table_risk_factors">
         <column>
          <property name="text">
           <string>Risk Factor Layer</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Weight (1-10)</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Correlation</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QPushButton" name="btn_add_factor">
         <property name="text">
          <string>Add Risk Factor Layer</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QPushButton" name="btn_generate_risk_map">
         <property name="styleSheet">
          <string notr="true">background-color: #4CAF50; color: white; font-weight: bold;</string>
         </property>
         <property name="text">
          <string>GENERATE FINAL RISK MAP</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_sampling">
      <attribute name="title">
       <string>2. Sampling Design</string>
      </attribute>
       <layout class="QVBoxLayout">
        <item><widget class="QLabel"><property name="text"><string>Module 2: Sampling Design - Coming Soon!</string></property></widget></item>
       </layout>
     </widget>
     <widget class="QWidget" name="tab_cost">
      <attribute name="title">
       <string>3. Cost Evaluation</string>
      </attribute>
      <layout class="QVBoxLayout">
        <item><widget class="QLabel"><property name="text"><string>Module 3: Cost-Effectiveness Evaluation - Coming Soon!</string></property></widget></item>
       </layout>
     </widget>
     <widget class="QWidget" name="tab_report">
      <attribute name="title">
       <string>4. Report & Export</string>
      </attribute>
      <layout class="QVBoxLayout">
        <item><widget class="QLabel"><property name="text"><string>Module 4: Reporting and Export - Coming Soon!</string></property></widget></item>
       </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>